dist/test_data.py

```diff
@@ -164,18 +165,21 @@ def __ge__(self, other):
 # Configuration that applies to the workload_generator component.
 #
 workload_generator = enabled_config_true + component_config + [
+    Config('custom_config', '',r'''
+        Config that specifies the number of user operation threads and their behaviour.''',
```

Comment:
Can we make this a bit more explicit? Something like "Config that specifies the number of custom_operation threads and their behaviour." The name "custom_config" seems a bit vague to me, so if we update the definition it'll help.

Reply:
Yep, thanks for the suggestion, see https://github.com/wiredtiger/wiredtiger/pull/7810/commits/6c119649ede04c59234496631b17521fe094ed98